prep_ubuntu.sh

```diff
@@ -51,8 +51,13 @@ upgrade_base_operating_system() {
   # now install it
   apt-get -y -q --no-install-recommends --force-yes install unzip iptables iptables-persistent
   systemctl enable netfilter-persistent
+
+  # allow root logon to SSH
   sed -i '/PermitRootLogin/d' /etc/ssh/sshd_config
   echo 'PermitRootLogin yes' >>/etc/ssh/sshd_config
+
+  # disable the power button
```

Comment:
Formerly, these were in a "power button" commit. It seems that that commit has gotten merged into this by accident.

Reply:
Fixed.